e2e-tests/common/methods.py

```diff
@@ -45,12 +47,16 @@ def analyzer_supported_entities(data):
     return response.status_code, response.content
 
 
-def redact(file, color_fill=None):
-    multipart_form_data = __get_multipart_form_data(file)
+def redact(file, color_fill=None, json_payload=False):
```

Comment:
nit: I suggest adding type hinting and docstrings.

Reply:
As this the just the test framework, I think we previously decided to not use both (aligned with the other tests)